### ISSUE TITLE
Display Stats/Reward Amounts for Deposits

### ISSUE THREAD

**Issue by jahabeebs** [open-dollar/od-app#241]
Part of TVL project https://github.com/open-dollar/od-app/issues/234

Depends on #240 

## Description
- Should display stats/reward amounts for deposits--data from nitro contracts

**Comment by jahabeebs** [comment]
Closing as won't do -- part of TVL project
